### ISSUE TITLE
[bug] Search box doesn't work (won't open the selected emoji)

### ISSUE THREAD

**Issue by manutero** [hfg-gmuend/openmoji.org#163]
check the video:

https://user-images.githubusercontent.com/2211052/159647105-e70324c4-7759-40b9-a268-9aa81dba4e7b.mov

Expected result: 
 - the selected emoji is opened.

***

Minor research:

 1. `XHR` request to `/data/openemoji.json.gz` returns 200 ✅ 
 2. `emoji-details-popup.js:112` logs an error 🔴 
    ```js
    // break if emoji wasn't found
    if (typeof currEmoji === 'undefined') {
      console.log("ERROR: Failed to find Emoji");
      return;
    }
    ```

https://user-images.githubusercontent.com/2211052/159647989-cf45193d-6fd6-4fdc-85f2-f2ccaa4a372e.mov

**Comment by b-g** [comment]
Hi @manutero, many thanks for reporting! We will have a look. cc @stephanbogner

**Comment by manutero** [comment]
More info: this is happening with Firefox, I've tried Chromium and it's able to open the emoji-detail view. 

My Firefox version: `98.0.1 (64 bits)` - the latest by the time I'm writing this

**Comment by manutero** [comment]
I've done some research. I've forked the project and launched it locally. Using firefox's developer console I've set some breakpoints and the behaviour is ... interesting:

1. **If I set a breakpoint, then resume the execution**: the emoji-detail view is opened just fine
2. **No breakpoint**: bugged behaviour (no detail view opened)

So there is some **`asynchronous issue`** around. 

I've recorded the following to support the **asynchronous issue**

This video is my `localhost:3000`


https://user-images.githubusercontent.com/2211052/160117429-705e3d5a-27e6-42f0-9150-84c4cf2d7087.mov

1. I click fox: nothing happens
2. I click another: fox is opened
